internal/lib/lib.go

```diff
@@ -419,26 +419,17 @@ func GetSubnetPrefixInt() int32 {
 }
 
 func GetNetworkNamesForVsVipNode() ([]string, error) {
```

Comment:
We can just use the GetVipNetworkList now. This function was intended to cover two different kinds of inputs.